packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -545,6 +545,7 @@ const TRANSFORMS = {
       startDelay: +obj.start_delay,
       startTime: metrics && toTimestamp(metrics.start_time),
       secureBoot: obj.platform.secureboot === 'true',
+      hvmBootFirmware: obj.HVM_boot_params?.firmware ?? 'bios',
```

Comment:
We already pass `boot: obj.HVM_boot_params` above, isn't it enough? Also, does it make sense to fallback to "bios"?